docs/Feature Documentation/Audio and Video Streaming/index.md

```diff
@@ -6,23 +6,18 @@ Before continuing, follow the [Install and Run Guide](../../getting-started/inst
 
 ### HMI Setup
 
-The Generic HMI does not currently support streaming.
+The [Generic HMI](https://github.com/smartdevicelink/generic_hmi) and [SDL HMI](https://github.com/smartdevicelink/sdl_hmi) both support streaming audio and some video formats in the browser using ffmpeg to transcode the video to VP8 WEBM. Instructions to install the required dependencies can be found in the HMI README:
```

Comment:
In reference to `using ffmpeg to transcode the video to VP8 WEBM`, does there need to be a note for audio as well (`... and the audio to <transcoded format>`)?

Reply:
commented that audio is transcoded to WAV